xapian-glib/xapian-utils.cc

```diff
@@ -131,3 +131,33 @@ xapian_revision (void)
 {
   return Xapian::revision ();
 }
+
+/**
+ * xapian_sortable_serialize:
+ *
+ * Convert a floating point number to a string, preserving sort order.
+ *
+ * This method converts a floating point number to a string, suitable for using
+ * as a value for numeric range restriction, or for use as a sort key.
+ */
+char *
+xapian_sortable_serialise (double value)
+{
+  return g_strdup (Xapian::sortable_serialise (value).c_str ());
+}
+
+/**
+ * xapian_sortable_unserialize:
+ *
+ * Convert a string encoded using sortable_serialise back to a floating point
+ * number.
+ *
+ * This expects the input to be a string produced by
+ * xapian_sortable_serialise(). If the input is not such a string, the value
+ * returned is undefined (but no error will be thrown).
+ */
```

Comment:
Same as `xapian_sortable_serialise`: needs annotations for the argument, the return value, and the API version.